lib/Conversion/XTenNNToTorch.cpp

```diff
@@ -163,6 +169,37 @@ std::optional<ValueRange> groupConv2dToTorch(GroupConv2dOp op, GroupConv2dOp::Ad
       ->getResults();
 }
 
+std::optional<ValueRange>
+convTranspose2dToTorch(ConvTransposeOp op, ConvTransposeOp::Adaptor adaptor,
+                       ArrayRef<Type> types, ValueRange values,
+                       ConversionPatternRewriter &rewriter) {
+  auto loc = op->getLoc();
+
+  auto newInput = values[0];
+  auto newWeights = values[1];
+  auto newBias = values[2];
+
+  auto stride = Torch::toTorchList(loc, rewriter, adaptor.getStride().vec());
+  auto dilation =
+      Torch::toTorchList(loc, rewriter, adaptor.getDilation().vec());
+  auto group =
+      rewriter.create<Torch::ConstantIntOp>(loc, adaptor.getGroupAttr());
+  auto outputPadding =
+      Torch::toTorchList(loc, rewriter, adaptor.getOutputPadding().vec());
+
+  // TODO: check the order is matched with what torch needs.
```

Comment:
Does it? 😄

Reply:
I believe it only supports symmetric padding since only two inputs are expected. Can you confirm @ehsan-toosi ?

Reply:
Haha that was supposed to be a TODO that I forgot to check. You guys did. It's implemented now.